pyEpiabm/pyEpiabm/tests/testing_parameters.json

```diff
@@ -486,6 +487,18 @@
             "distancing_house_susc": 0.8,
             "distancing_place_susc": [0.8, 0.8, 0.8, 0.8, 0.8, 0.8],
             "distancing_spatial_susc": 0.8
+        },
+        "testing": {
+            "start_time": 6,
+            "policy_duration": 365,
+            "case_threshold": 0,
+            "testing_capacity": [2, 2],
+            "false_positive": [0.1, 0.1],
+            "false_negative": [0.1, 0.1],
```

Comment:
Could we get the real parameters here - I think we know these quite well and it would be cool to get an idea of false vs true positives!